ow_lander/README.md

```diff
@@ -1,84 +1,149 @@
-The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+> The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
```

Comment:
Is the '>' a typo or this Markdown syntax?

Reply:
I now see that this section appears like an email quote, which looks odd to me...

Reply:
It is a Markdown syntax. On my markdown viewer and github the paragraph would appear grayed out.